python/build-wheel-macos.sh

```diff
@@ -8,7 +8,7 @@ set -x
 
 DOWNLOAD_DIR=python_downloads
 
-NODE_VERSION="14"
+NODE_VERSION="20"
```

Comment:
same here.. maybe just drop using nvm. not a fan of using nvm in build procedures.

one can actually just create a tmp dir, download and extract node, set `PATH`, and build the dashboard I think.

optional though.